Clinical-Genomics/scout
# Fix case inactivation on update

Fix #1633

**How to test**:
1. Make sure that editing case age and tissue type (also with empty case age) doesn't cause crash
1. Make sure that if a case is active it doesn't become inactive upon change (for instance modifying the above age or tissue type)

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN, CR
